pkg/registry/controller/etcd/etcd.go

```diff
@@ -32,8 +32,24 @@ type REST struct {
 }
 
 // NewREST returns a RESTStorage object that will work against replication controllers.
-func NewREST(s storage.Interface) (*REST, *StatusREST) {
+func NewREST(s storage.Interface, useCacher bool) (*REST, *StatusREST) {
 	prefix := "/controllers"
+
+	storageInterface := s
```

Comment:
Wow, can we extract this code to a function somewhere? I'm really not a fan of all this duplication.